### DISCUSSION TITLE
I received "Failed to create event handler" error when I want to create click event. Also can't access element name in MainPage.cs

### DISCUSSION THREAD

**Discussion by haghanihakimi** [unoplatform/uno#8390] [Q&A]
Recently I tried to create a sample app using UNO. It is sample because I want to get some experience with UNO framework. I don't have any problem with UWP without UNO but when I use UNO, first, I cannot create event handler. When I click on "`<New Event Handler>`" it throws an error which says: "**Failed to create event handler**", without more details or information about mentioned error.
So I have to do it manually, still, when I create event handler manually, now I cannot access element name from **MainPage.cs**! For example, if there is an element like this:
`<TextBox x:Name="result" IsReadOnly="True" HorizontalAlignment="Center" Margin="0,85,0,0" TextWrapping="Wrap" Text="Results" VerticalAlignment="top" Width="240" />`
I cannot access **result** from MainPage.cs, there errors which say "The name **result** does not exist in the current context".
I am not sure what is missing in my UNO UWP project, but as I followed articles and a video on YouTube, it should work but it doesn't!
Anyone knows what I missed or what is wrong with this project?
If there is any more information that you need and I forgot to give, please let me know.
If any one knows the answer please answers below. Thank you.

**Comment by jeromelaban** [comment]
Thanks for the report, see here: https://platform.uno/docs/articles/get-started-wizard.html#event-handler-cannot-be-added-automatically

**Reply by haghanihakimi** [reply]
Thank you yeah that worked.
